lib/gpu/lib/gpu.dart

```diff
@@ -11,7 +11,12 @@
 ///  * [Flutter GPU Wiki page](https://github.com/flutter/flutter/wiki/Flutter-GPU).
 library flutter_gpu;
 
+import 'dart:ffi';
+import 'dart:nativewrappers';
+import 'dart:typed_data';
+
 export 'src/smoketest.dart';
 
-export 'src/context.dart';
-export 'src/buffer.dart';
+part 'src/formats.dart';
+part 'src/context.dart';
+part 'src/buffer.dart';
```

Comment:
I switched back to this pattern due to circular dependencies. GpuContext depends on DeviceBuffer, DeviceBuffer depends on GpuContext. The constructors for DeviceBuffer are locked down anyhow, so we can remove the DeviceBuffer->GpuContext dependency, but would prefer to do that as follow-up work. These bindings are an enormous pain as-is. 😅

Reply:
Just as an FYI, circular imports are A-OK in Dart.

Reply:
Hmm, maybe what I was actually struggling with was getting imports working at all between files in the package. I'm guessing my import URIs were wrong.